src/compiler/debug.ts

```diff
@@ -1126,7 +1126,7 @@ m2: ${(this.mapper2 as unknown as DebugTypeMapper).__debugToString().split("\n")
 
         function renderGraph() {
             const columnCount = columnWidths.length;
-            const laneCount = nodes.reduce((x, n) => Math.max(x, n.lane), 0) + 1;
+            const laneCount = ts.maxBy(nodes, 0, n => n.lane) + 1;

```

Comment:
Do the import here rather than using `ts`; it's only in scope because https://github.com/microsoft/TypeScript/issues/51530